src/AccessibilityInsights.Rules/CCAControlTypesFilter.cs

```diff
@@ -6,6 +6,9 @@
 
 namespace Axe.Windows.Rules
 {
+    /// <summary>
+    /// Remove this class from Rules assembly
```

Comment:
Want to double-check the added comment here about removing the class from the Rules assembly. Would it be better to omit this and track the follow-up in a different way?